### DISCUSSION TITLE
running node demo require Node version 14 or above

### DISCUSSION THREAD

**Discussion by uzair004** [vladmandic/human#135] [General]
running node demo using Node version 13 or below return error

### Error: 
SyntaxError: Unexpected token '.'

### Error Reason: 
error is because of Optional Chaining used which is not supported in NodeJS 13 or below. 
you can confirm here [Optional Chianing MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) or check this [StackOverflow question](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining)

Minimum Node version require should be bumped to 14 or above instead of 12 as mentioned in documentation.

**Comment by vladmandic** [comment]
Thanks - I'll make the updates when I'm back in ~10 days.

**Comment by vladmandic** [comment]
updated requirements to nodejs v14
